trigger/task.go

```diff
@@ -76,7 +76,12 @@ func newAlertRecord(subID string, t *task.Task, alertType string) *alertrecord.A
 	}
 }
 
-func taskFinishedTwoOrMoreDaysAgo(taskID string) (bool, error) {
+func taskFinishedTwoOrMoreDaysAgo(taskID string, sub *event.Subscription) (bool, error) {
+	renotify, found := sub.TriggerData[event.RenotifyIntervalKey]
+	renotifyInterval, err := strconv.Atoi(renotify)
+	if renotify == "" || err != nil || !found {
```

Comment:
Should we support a value for never re-notify, e.g., -1?

Reply:
I think in that scenario you would set a really large value for this

Reply:
I like -1 better?